var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -47,7 +47,10 @@ class PyNumpy(PythonPackage):
 
     # FIXME: numpy._build_utils and numpy.core.code_generators failed to import
     # FIXME: Is this expected?
-
+    
```

Comment:
This line still has spaces at the beginning. You should be able to configure your text editor to automatically remove trailing whitespace.